app/helpers/layout_helper.rb

```diff
@@ -63,6 +63,20 @@ def title(page_title, page_header = nil)
     @page_header ||= page_header || page_title.to_s
   end
 
+  def edit_title(obj = nil, page_header: nil, attrbiture: nil)
+    return title(_('Edit', page_header)) unless obj
+
+    valid_obj = obj
+    label_attr = attrbiture || 'to_s'
+
+    unless obj.valid?
+      # we need all attributes and then we need to override changed values to old ones
+      old_attributes = obj.attributes.merge(obj.changed_attributes)
```

Comment:
I might be missing something here but I don't the comment correctly describes the behavior. You take existing (uodated) attributes and merge chages to them. Then you build new object with new attributes. Of course the object is valid since no `valid?` or `save` was called on it. But imho it has the same values of attributes. Could maybe a bit better explain what is the cause and why this helps? Thanks